Sources/Smithy/Document/SmithyDocument.swift

```diff
@@ -8,8 +8,9 @@
 import struct Foundation.Data
 import struct Foundation.Date
 
-public protocol SmithyDocument {
+public protocol SmithyDocument: Sendable {
```

Comment:
All of the `SmithyDocument` implementations are now sendable without any code change, since they only store data that is either immutable and/or `Sendable`.